e2e/host/zonefile

```diff
@@ -47,4 +47,12 @@ chaining.path.path.example.com.                        IN A 172.20.10.2
 chained.host.path.example.com.                         IN A 172.20.10.2
 _redirect.chaining.path.path.example.com.              IN TXT "v=txtv0;type=path;to=https://fallback-to.path.path.example.com;code=302"
 _redirect.releases.chaining.path.path.example.com.     IN TXT "v=txtv0;type=path;to=http://chained.host.path.example.com"
-_redirect.chained.host.path.example.com.               IN TXT "v=txtv0;type=host;to=https://chained-redirect.host.path.example.com;code=302"
\ No newline at end of file
+_redirect.chained.host.path.example.com.               IN TXT "v=txtv0;type=host;to=https://chained-redirect.host.path.example.com;code=302"
+
+apex.host.host.example.com.                            IN A 172.20.10.2
+_redirect._.apex.host.host.example.com.                IN TXT "v=txtv0;type=host;to=https://apex-to.host.host.example.com;code=302"
+
+apexuse.host.host.example.com.                         IN A 172.20.10.2
+userec.host.host.example.com.                          IN A 172.20.10.2
```

Comment:
Why the need to add an IP to userec? Shouldn't the TXT record suffice?

Reply:
Leftovers :sweat_smile: